README.md

```diff
@@ -1,2 +1,25 @@
-# object_detection_example_structure
-Example of folder structure of a complete run on DeepSportRadar Instance Segmentation Challenge dataset
+### Object Detection example on a selection on SportRadar Challenge
+
+This is an example of folder structure of a complete run on [DeepSportRadar Instance Segmentation Challenge](https://github.com/DeepSportRadar/instance-segmentation-challenge)!
+Predictions are computed on the union of val and test sets using a detectron2 mask_rcnn_R_50_FPN_3x model finetuned on the train set. Metadata are created with the name of the folders in which images are contained.
+
```

Comment:
I'd drop here a rough folder structure as a reference such that users get an idea what to expect. You can use tree.